arch/arm/src/stm32f0l0g0/stm32_adc.c

```diff
@@ -1132,6 +1139,16 @@ static void adc_common_cfg(struct stm32_dev_s *priv)
   uint32_t clrbits = 0;
   uint32_t setbits = 0;
 
+#if defined(CONFIG_STM32F0L0G0_STM32G0)
+
+  /* G0 series maps CCR inside ADC1; access directly */
+
+  clrbits |= ADC_CCR_PRESC_MASK | ADC_CCR_VREFEN | \
+             ADC_CCR_TSEN | ADC_CCR_VBATEN;
+  setbits |= ADC_CCR_PRESC_4 | ADC_CCR_VREFEN;
```

Comment:
why hardcode PRESC_4 and always enable VREFEN? Can we select these automatically like in stm32h7 or stm32h5?